src/cordova/build_plugin.action.mjs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from 'url';
+
+import {cpSync, mkdirSync} from 'fs';
+import path from 'path';
+import rmfr from 'rmfr';
```

Comment:
*discussion* it seems we have references to packages with duplicated functionalities, like `rmfr` here is duplicated with `rimraf`.